Source/Core/DolphinQt/NetPlay/NetPlayBrowser.cpp

```diff
@@ -127,6 +124,8 @@ void NetPlayBrowser::CreateWidgets()
   m_button_box->button(QDialogButtonBox::Ok)->setEnabled(false);
 
   setLayout(layout);
+
+  RestoreSettings();
```

Comment:
Just curious, why was this moved?  It seems odd for it to be in `CreateWidgets`

Reply:
It's a minor thing, but I guess I should have mentioned it, or at least added a comment.

`RestoreSettings()` set new values for the widgets, and those widgets trigger `Refresh()` when changed. The idea is that settings need to be restored *before* `ConnectWidgets()`, or else a bunch of `Refresh()` calls will be made (although only the last one will probably win).

I argued with myself whether the `RestoreSettings()` call should be placed after `CreateWidgets()`, or inside `CreateWidgets()`. I thought that, if `CreateWidgets()` was ever to be called again, then `RestoreSettings()` would need to be called again as well, so having it inside makes sense.

However, the same argument could be used for having `ConnectWidgets()` inside `CreateWidgets()`, which we don't have. So, for consistency, `RestoreSettings()` should be invoked directly from the constructor, before `ConnectWidgets()`.

I'll update the PR.